pkg/kubectl/resource/builder.go

```diff
@@ -532,34 +532,30 @@ func (b *Builder) visitorResult() *Result {
 		if len(b.resources) == 0 {
 			return &Result{singular: isSingular, err: fmt.Errorf("you must provide a resource and a resource name together")}
 		}
-		if len(b.resources) > 1 {
-			return &Result{singular: isSingular, err: fmt.Errorf("you must specify only one resource")}
-		}
 
+		visitors := []Visitor{}
```

Comment:
I would recommend instead that when you have `a,b,c d` in ResourceTypeOrName that you convert them to resource tuples (which already handle this case).  Then you can remove this entire block (I think tuples is a strict superset of names at this point).